gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -3451,7 +3451,7 @@ subroutine Run ( gc, import, export, clock, rc )
   ! swath only
    epoch_swath_grid_case: do n=1,nlist
       call MAPL_TimerOn(GENSTATE,trim(list(n)%collection))
-      call MAPL_TimerOn(GENSTATE,"SwathGen")
+      call MAPL_TimerOn(GENSTATE,"SwathRun")
```

Comment:
What does "Run" mean here?    You are already inside of the Run method of history.  Ideally there would be some more specific term.    How about "SwathRegridAccum"  of if that feels too long,  "SwathRgrdAcc"?

Reply:
Looking at some of the other work below, maybe what you want here is 2 timers.
1. "Swath" for context
2. "RegridAccum" as the nested timer for this specific action, as SwatchRun appears again later.